File: Condicionales/Ejercicio7-Condicionales.cpp
/*7. Escriba un programa que solicite una edad (entero) e indique en la salida estándar si la edad introducida está en el rango [18-25].*/

#include<iostream>
using namespace std;
int main(){
    int edad;
    cout<<"Digite su edad: "; 
    cin>>edad;

    //18-19-20-21-22-23-24-25
    if (edad>=18 && edad<=25){
        cout<<"Su edad esta en el rango de [18-25] ";
    }
    else {
        cout<<"Su edad no esta en dicho rango";
    }
    return 0;
} 

File: Comparativa Python/Ejercicio7-Condicionales.py
# 7. Escriba un programa que solicite una edad (entero) e indique en la salida
# estándar si la edad introducida está en el rango [18-25].

edad = int(input('Digite edad: '))

if edad >= 18 and edad <= 25:
    print('La edad está dentro del rango [18-25]')
else:
    print('La edad está FUERA del rango [18-25]')
